projects/packages/forms/src/contact-form/libs/date-picker/date-picker.css

```diff
@@ -0,0 +1,430 @@
+	border: 0;
+	background: var( --dp-ui-background );
+	color: var( --dp-ui-color );
+}
+
+.dp-permanent .dp-close,
+.dp-permanent .dp-clear {
+	display: none;
+}
+
+.dp-close:active,
+.dp-clear:active,
+.dp-today:active,
+.dp-next:active,
+.dp-prev:active,
+.dp-cal-month:active,
+.dp-cal-year:active {
+	background: var( --dp-ui-background-selected);
+	color: var( --dp-ui-color-selected );
+}
+
+@media screen and (min-device-width: 1200px) {
+
+	.dp-close:hover,
+	.dp-close:focus,
+	.dp-clear:hover,
+	.dp-clear:focus,
+	.dp-today:hover,
+	.dp-today:focus,
+	.dp-next:hover,
+	.dp-next:focus,
+	.dp-prev:hover,
+	.dp-prev:focus,
+	.dp-cal-month:focus,
+	.dp-cal-month:hover,
+	.dp-cal-year:hover,
+	.dp-cal-year:focus {
+		background: var( --dp-ui-background-selected);
+		color: var( --dp-ui-color-selected );
+	}
+}
+
+.dp-col-header,
+.dp-day {
+	display: inline-block;
+	padding: 0;
+	text-align: center;
+	height: 36px;
+	border-radius: 50%;
+	width: 36px;
+}
+
+.dp-focusable,
+.dp-day {
+	cursor: pointer;
+}
+
+.dp-days {
+	display: grid;
+	align-content: space-between;
+	grid-template-columns: repeat(7, 1fr);
+	grid-template-rows: auto auto auto auto auto auto auto;
+	column-gap: 1px;
+	row-gap: 1px;
+}
+
+.dp-col-header {
+	text-transform: uppercase;
+	color: var(--dp-text-color);
+	font-weight: var(--dp-bold);
+	line-height: 36px;
+}
+
+.dp-month {
+	font-size: 16px;
+	padding: 0;
+}
+
+.dp-year {
+	display: block;
+	padding: 8px 40px;
+	width: 100%;
+	font-size: 16px;
+}
+
+.dp-edge-day {
+	cursor: default;
+}
+
+.dp .dp-edge-day {
+	color: var(--dp-date-edge-color);
+}
+
+.dp-current:focus,
+.dp-day:focus,
+.dp-month:focus,
+.dp-selected:focus,
+.dp-year:focus {
+	outline: none;
+	box-shadow: none;
+}
+
+.dp .dp-current:focus,
+.dp .dp-day:focus,
+.dp .dp-month:focus,
+.dp .dp-selected:focus,
+.dp .dp-year:focus {
+	background: var(--dp-date-background-focus);
+	color: var(--dp-date-color-focus);
+}
+
+.dp-day:hover,
+.dp-month:hover,
+.dp-year:hover,
+.dp-selected:hover {
+	box-shadow: none;
+}
+
+.dp .dp-selected {
+	background: var(--dp-date-background-selected);
+	color: var(--dp-date-color-selected);
+}
+
+.dp .dp-current {
+	background: var(--dp-date-background-selected);
+	color: var(--dp-date-color-selected);
+}
+
+.dp .dp-day:hover,
+.dp .dp-month:hover,
+.dp .dp-year:hover,
+.dp .dp-selected:hover {
+	background: var(--dp-date-background-hover);
+	color: var(--dp-date-color-hover);
+}
+
+.dp .dp-day-disabled {
+	background: var(--dp-date-background-disabled);
+	color: var(--dp-date-color-disabled);
+}
+
+.dp-day-disabled:focus,
+.dp-day-disabled:hover {
+	background: var(--dp-date-background-disabled);
+}
+
+.dp-focuser {
+	position: absolute;
+	z-index: 0;
+	top: 50%;
+	left: 50%;
+}
+
+/* Responsive overrides */
+@media (max-width: 480px), (max-height: 480px) {
+
+	.dp-modal .dp {
+		font-size: 0.9em;
+		width: auto;
+		width: 100%;
+	}
+
+	.dp-day-of-week,
+	.dp-day {
+		padding: 8px;
+	}
+}
+
+@keyframes slide-up {
```

Comment:
Too generic animation name; it'll end up in global scope at the site.

Reply:
good call.